packages/api/scripts/cli.js

```diff
@@ -49,7 +49,6 @@ prog
         VERSION: JSON.stringify(version),
         COMMITHASH: JSON.stringify(git.long(__dirname)),
         BRANCH: JSON.stringify(git.branch(__dirname)),
-        ENV: opts.env || 'dev',
```

Comment:
What was the reasoning to remove this? Can't we keep it?
We need this to distinguish the env where the error happens (prod or staging)

Reply:
this is the `define` config for esbuild. It tells it to replace strings in the built bundle... our "env" value should come from the env vars passed to miniflare or cloudflare, rather than being baked into the bundle.

Reply:
I thought this was needed in the bundle for https://docs.sentry.io/product/sentry-basics/environments/ but yeah, I am not sure on it

Reply:
Ok, I see it is likely the end line with `.newDeploy()`

Reply:
yep, sentry can pick the env out of the version string, or we also pass it to the `newDeploy` call explicitly. Then at runtime, env.ENV should have been set in Cloudflare as a runtime env var, which Sentry sets on it's requests to their api when flagging errors, so reporting is logged under the correct build.

Reply:
hmm... what is not clear yet is how/if we are setting the env.ENV correctly at build time... I don't see any evidence of that in https://github.com/web3-storage/web3.storage/blob/main/.github/workflows/api.yml yet

Reply:
via wrangler.toml file?

Reply:
check this

```console
❯ wrangler build --env production
🌀  Running npm run build

> @web3-storage/api@5.1.4 build
> node scripts/cli.js build

{ 'opts.env': 'dev', 'process.env.ENV': undefined }

❯ wrangler build --env oli
🌀  Running npm run build

> @web3-storage/api@5.1.4 build
> node scripts/cli.js build

{ 'opts.env': 'dev', 'process.env.ENV': undefined }
```

Reply:
i think `opts.env` as passed to the build cli is always `dev` and that's why we're seeing all our releases listed as `dev` on sentry

<img width="424" alt="Screenshot 2022-02-23 at 11 43 46" src="https://user-images.githubusercontent.com/58871/155313275-247e7a71-6c2a-441f-9d52-e3f623b8445b.png">

Reply:
the env gets passed in explicitly in nft.storage by calling a different build command per env

https://github.com/nftstorage/nft.storage/blob/c47f94fc267c80279791f82d60a5ee031847b87e/packages/api/wrangler.toml#L66-L68